apps/client-web/src/docs/pages/DocumentPage.tsx

```diff
@@ -21,13 +21,14 @@ interface DocumentPageProps {
 export const DocumentPage: React.FC<DocumentPageProps> = ({ mode }) => {
   const docMeta = useDocMeta()
   const navigate = useNavigate()
+  const latestLoading = useRef(true)
```

Comment:
we should use `useState` here and

```
useEffect(() => {
  if (!loading) setLatestLoading(false)
} , [loading])
```

Variables participated in conditional rendering should always be a `state` instead of a `ref`.